cortex.yaml

```diff
@@ -22,6 +22,10 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 279ed566d01228e4b95fd628a3d79dd3ca95c9d8
 
+  - uses: patch
+    with:
+      patches: go-grpc.patch
+
```

Comment:
Given the size of the patch, I'd love if we could also add a quick test to that package!